Makefile

```diff
@@ -9,7 +9,37 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+GOOS ?= $(shell go env GOOS)
+GOARCH = amd64
+BUILD_DIR ?= ./out
+
+%.exe: %
+	mv $< $@
+
+$(BUILD_DIR):
+	mkdir -p $(BUILD_DIR)
+
+ORG := github.com/grafeas
+PROJECT := kritis
+RESOLVER_PROJECT := resolver
```

Comment:
In the plugin.yaml I named this binary [resolve-tags](https://github.com/grafeas/kritis/blob/master/cmd/kritis/kubectl/plugins/resolve/plugin.yaml#L3). I think resolver sounds good too, but they should probably match.